dns-record-check/README.md

```diff
@@ -0,0 +1,35 @@
+# Chainlink External Adapter to Check DNS Records
```

Comment:
This `dns-record-check` is not a standalone adapter, it needs to use the underlying `dns-query` adapter to complete its work. We call these types of adapters composite adapters and currently we group them in `/composite`.

We do this just to make it explicit, for now. In the future, we may accept that almost all adapters are some kind of composite and as such may stop grouping them in this way.

For now, I would prefer if we stay explicit and group these kinds of adapters in `/composite`.